docker/compose.yml

```diff
@@ -88,29 +96,17 @@ services:
         max-file: "10"
 
 
-  beat:
-    depends_on:
-      - rabbitmq
+  scheduler:
     deploy:
       restart_policy:
         condition: on-failure
         max_attempts: 3
-    image: "${DOCKER_IMAGE_NAMESPACE}/taranis-worker:${TARANIS_TAG:-latest}"
-    command: "--app worker beat"
+    ports:
+      - "8000:8000"
+    image: "${DOCKER_IMAGE_NAMESPACE}/taranis-scheduler:${TARANIS_TAG:-latest}"
```

Comment:
`TARANIS_TAG` can't be used here. At the moment the versioning is incompatible since the scheduler image is 0.3.X at the moment.